packit_service/worker/tasks.py

```diff
@@ -74,3 +113,240 @@ def babysit_copr_build(self, build_id: int):
+def run_pr_comment_copr_build_handler(
+    event: dict, package_config: dict, job_config: dict
+):
+    handler = GitHubPullRequestCommentCoprBuildHandler(
+        package_config=load_package_config(package_config),
+        job_config=load_job_config(job_config),
+        data=EventData.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_push_copr_build_handler")
+def run_push_copr_build_handler(event: dict, package_config: dict, job_config: dict):
+    handler = PushCoprBuildHandler(
+        package_config=load_package_config(package_config),
+        job_config=load_job_config(job_config),
+        data=EventData.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_installation_handler")
+def run_installation_handler(event: dict, package_config: dict, job_config: dict):
+    handler = GithubAppInstallationHandler(
+        package_config=None,
+        job_config=None,
+        data=None,
+        installation_event=InstallationEvent.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_testing_farm_handler")
+def run_testing_farm_handler(
+    event: dict, package_config: dict, job_config: dict, chroot: str
+):
+    handler = GithubTestingFarmHandler(
+        package_config=load_package_config(package_config),
+        job_config=load_job_config(job_config),
+        data=EventData.from_event_dict(event),
+        chroot=chroot,
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_testing_farm_comment_handler")
+def run_testing_farm_comment_handler(
+    event: dict, package_config: dict, job_config: dict
+):
+    handler = GitHubPullRequestCommentTestingFarmHandler(
+        package_config=load_package_config(package_config),
+        job_config=load_job_config(job_config),
+        data=EventData.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_testing_farm_results_handler")
+def run_testing_farm_results_handler(
+    event: dict, package_config: dict, job_config: dict
+):
+    tests = event.get("tests")
+    result = TestingFarmResult(event.get("result")) if event.get("result") else None
+    pipeline_id = event.get("pipeline_id")
+    log_url = event.get("log_url")
+    copr_chroot = event.get("copr_chroot")
+    message = event.get("message")
+
+    handler = TestingFarmResultsHandler(
+        package_config=load_package_config(package_config),
+        job_config=load_job_config(job_config),
+        data=EventData.from_event_dict(event),
+        tests=tests,
+        result=result,
+        pipeline_id=pipeline_id,
+        log_url=log_url,
+        copr_chroot=copr_chroot,
+        message=message,
+    )
```

Comment:
Same as above. Please, avoid one-time variables and, ideally, avoid getters at all.